tests/cupy_tests/random_tests/test_generator.py

```diff
@@ -19,15 +19,16 @@ class RandomGeneratorTestCase(unittest.TestCase):
     target_method = None
 
     def setUp(self):
-        self.rs = generator.RandomState(seed=testing.generate_seed())
+        self.__seed = testing.generate_seed()
```

Comment:
Why do you use double underscore?

Reply:
Because a child class does not need to know the seed to reset the random state.

Reply:
OK